lib/internal/webstreams/compression.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const {
+  ObjectDefineProperties,
+  Symbol,
+} = primordials;
+
+const {
+  codes: {
+    ERR_INVALID_ARG_VALUE,
+    ERR_INVALID_THIS,
+  },
+} = require('internal/errors');
+
+const {
+  newReadableWritablePairFromDuplex,
+} = require('internal/webstreams/adapters');
+
+const {
+  customInspect,
+  kEnumerableProperty,
+} = require('internal/webstreams/util');
+
+const {
+  customInspectSymbol: kInspect,
+} = require('internal/util');
+
+const {
+  createDeflate,
+  createInflate,
+  createGzip,
+  createGunzip,
+} = require('zlib');
```

Comment:
Can you please lazy-load this on first use?

Reply:
Done!

Reply:
Does this still matter a lot now that we have code cache / snapshots?